cas-server-webapp-support/src/main/java/org/jasig/cas/web/support/CookieRetrievingCookieGenerator.java

```diff
@@ -103,9 +103,14 @@ public void addCookie(final HttpServletRequest request, final HttpServletRespons
      * @return the cookie value
      */
     public String retrieveCookieValue(final HttpServletRequest request) {
-        final Cookie cookie = org.springframework.web.util.WebUtils.getCookie(
-                request, getCookieName());
-        return cookie == null ? null : this.casCookieValueManager.obtainCookieValue(cookie, request);
+        try {

```

Comment:
This change is not related to this PR, isn't it? I guess there was already a fix to trap encoding issue...